tunix/examples/data/translation_dataset.py

```diff
@@ -89,11 +89,50 @@ def tokenize(
     return np.array(int_list, dtype=np.int32)
 
 
+class Gemma3Tokenizer(spm.SentencePieceProcessor):
```

Comment:
Do we need this? Can't we just use the GemmaTokenizer and pass the correct SPM file path?